components/mailer.rst

```diff
@@ -16,8 +16,127 @@ Installation
 
 .. include:: /components/require_autoload.rst.inc
 
+
+Introduction
+------------
+
```

Comment:
Hey @weaverryan , @wouterj  here we need some introduction. Do you guys have some good text for here ?

This is what I have been thinking. 

`Symfony mailer is an experimental component introduced in 4.3. This component will eventually replace swiftmailer.`

Reply:
what about : This component aims to replace swiftmailer?